benchmark/CMakeLists.txt

```diff
@@ -32,18 +32,44 @@ target_link_libraries(ubench
 	pthread
 	m)
```

Comment:
The message in line no. 6 should be updated:
```
message(WARNING "The ubench SHOULD NOT be run in the Debug build type!")
```

Reply:
Done.

Reply:
Not done

Reply:
It can be, for example: "The **benchmarks** SHOULD NOT be run in the Debug build type!":
```
message(WARNING "The benchmarks SHOULD NOT be run in the Debug build type!")
```

Reply:
Done.